app/models/customer.py

```diff
@@ -0,0 +1,39 @@
+from flask import current_app
+from app import db
+from sqlalchemy import Table, Column, Integer, ForeignKey
+# from sqlalchemy.orm import relationship, back_populate
+# from app.models.video import Video
+
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True) 
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    created = db.Column(db.DateTime, nullable=True, default=None)
+    videos_checked_out_count = db.Column(db.Integer, default=0, nullable=False)
+    videos = db.relationship("Rental", back_populates="customer")
```

Comment:
👍  I think this is a good solution to stick with. I would consider adding `lazy=True` in the future as it makes it a little easier to query data that has relationships connected to it. `backref` allows you to  relational data from either side. So you can use customer.videos OR, with backref, you can use video.customer to get all the customers connected to the video by rental. 

Based on what I know `backref` only needs to be used in one or the other model. Whereas `back_populates` needs to be used in both tables to connect the relationship more easily